depmapomics/mutations.py

```diff
@@ -781,6 +776,65 @@ def addLikelyLoF(row, vep_col="vep_impact", oncoimpact_col="oncokb_effect"):
         return False
 
 
+def addRescueReason(maf, rescue_reason_colname="rescue_reason"):
+    """add column to indicate why variants are rescued
+
+    Args:
+        maf (df): MAF-like dataframe containing variants
+        rescue_reason_colname (str): name of the rescue reason column
+
+    Returns:
+        maf (df): MAF-like dataframe containing variants, with rescue reason"""
+    # initialize column as empty lists
+    maf[rescue_reason_colname] = [[] for _ in range(maf.shape[0])]
+
+    # go over each possible rescue reason in vcf_to_depmap and append them to the list
+    maf.loc[
+        (maf["oncokb_effect"].isin(["Loss-of-function", "Gain-of-function"]))
```

Comment:
Should we also allow `Likely Loss-of-function` and `Likely Gain-of-function` values?